Include/internal/pycore_gc.h

```diff
@@ -238,14 +244,20 @@ static inline void _PyObject_GC_TRACK(
                           "object is in generation which is garbage collected",
                           filename, lineno, __func__);
 
-    PyInterpreterState *interp = _PyInterpreterState_GET();
-    PyGC_Head *generation0 = &interp->gc.young.head;
+    PyThreadState *tstate = _PyThreadState_GET();
+    struct _gc_runtime_state *gcstate = &tstate->interp->gc;
```

Comment:
```suggestion
    struct _gc_runtime_state *gcstate = &_PyInterpreterState_GET()->gc;
```

Maybe it'll be better to use the same expression as below?
There's no another call of `_PyThreadState_GET` in this file.